### DISCUSSION TITLE
Duvida

### DISCUSSION THREAD

**Discussion by Sakay5736** [Zian25/UniTools-Termux#10] [Q&A]
Bom eu instalei a ferramenta de senha do Facebook agora poderia me falar qual código para executar a ferramenta

**Comment by Zian25** [comment]
Olá, a ferramenta está na pasta 'root', você pode acessar pelo comando:

cd ~

Depois utilize: cd nomeDaFerramenta

Para inicializar é dificil dar alguma explicação, preciso saber o nome da ferramenta pois é necessário saber qual linguagem ela está escrita para iniciar...
